### PR TITLE
Progress Report #37: Love, Geoff, and Robots

### PR DESCRIPTION
**Pull request by ike709**
TODO:

- [x] Write the thing
- [x] Name replacements
- [x] Patrons
- [x] Title
- [x] Pics/videos
- [x] Thumbnail
- [x] Write the short intro bit
- [x] Compression script
- [x] Any other TODOs (date and thumbnail)
- [x] Review

### CODE REVIEW

File: content/post/22-09-09-progress-report-37.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
++++
+title = "TODO"
```
Comment (ike709) [inline content/post/22-09-09-progress-report-37.md:2]:
```suggestion
title = "Progress Report #37: Love, Geoff, and Robots"
```

File: content/post/22-09-09-progress-report-37.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+- Reduced PKA recharge time *Contributed by mirrorcult*
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+TODO: Pulse stats
+*13spacemen, 20kdc, Ablankmann, Aerocrux, AJCM-git, asperger-sind, BismarckShuffle, ChilbroBaggins, corentt, CormosLemming, Deahaka, Delete69, DoctorBeard, DogZeroX, drakewill-CRL, Duddino, ElectroSR, Rane, Emisse, EmoGarbage404, fooberticus, FunkySphere, GoodWheatley, HoofedEar, hubismal, ike709, jacksonzck, jessicamaybe, jproads, keronshb, KIBORG04, Kimpes, lapatison, Level10Cybermancer, metalgearsloth, Mirino97, mirrorcult, ModeratelyAware, moony, Morb0, Myctai, notafet, OldDanceJacket, Pangogie, PaulRitter, Peptide90, PJB3005, Rinkashikachi, sBasalto, ShadowCommander, sunbear-dev, TaralGit, TekuNut, themias, TimrodDX, vulppine, wrexbe, Ygg01, Daemon, zerorulez, Zumorica*
+
+### Patrons
+TODO
```
Comment (ike709) [inline content/post/22-09-09-progress-report-37.md:358]:
```suggestion
Matouš Hrdlička, Roman, Jacob Schramm, Nathan Zaldivar, SpydAxe, BokChoy, Nicholas Williamson, KevKev, Gothryd, Brandon Campbell, Matthew C Miklaucic, Serianas, lapatison, Uinseann, osborn, LPGaming, Alex Fry, Ramiro Agis, Mikhail, liltenhead, James Andrew Peoples Jr, Serathis, Joel, Kris Piper, Phillip Inman, Jex N, Peter "Azmond" Newhouse, Seija, Hamcha, Oxyclean114, Cormos Lemming, grhmhome, Spiffyfishface, Callum Tubrett, John Edward Hamilton Barchard, Wrexbe, Austin Nelson, AquaDraco, hh, Nicholas Hillblom, Florian, Viridian, Daskata, TacoCub, Jack Rose, The Hateful Flesh, Will M., spinnermaster, Koyki, Dan Griskardian, Robin Rottstock, Altana, Durp, Joshington Awesomahee, Eric VW, Evan Armstrong, Mono, Ethan Keller, DramaBuns, KrystalDisc, Darren Brady, Zandario, Anthony Fleck, vifs_vestige, Bobberunio, CPM311, ThatGuyGW, Farewell Fire, Daniel Thompson, Tomeno
```

File: content/post/22-09-09-progress-report-37.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+- Miasma is no longer visible until it's a problematic amount *Contributed by Rane*
+- Fixed various visual bugs for held/equipped items *Contributed by ElectroSR*
+- Syndie shoulder holsters now accept melee weapons too *Contributed by Myctai*
+- Fixed some helmets being invisible *Contributed by ElectroSR*
+- Reduced PKA recharge time *Contributed by mirrorcult*
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+TODO: Pulse stats
```
Comment (ike709) [inline content/post/22-09-09-progress-report-37.md:353]:
```suggestion
```

**Review by DrSmugleaf — APPROVED** [review]

File: content/post/22-10-27-progress-report-37.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+title = "Progress Report #37: Love, Geoff, and Robots"
+author = "DrSmugleaf, ike709 & Visne"
+date = 2022-10-27 # TODO Update
+categories = [
+    "Progress Report"
+]
+reddit = ""
+thumbnail = "pr_37"
++++
+Better late than never (TODO: write something better.)
```
Comment (moonheart08) [inline content/post/22-10-27-progress-report-37.md:11]:
```suggestion
Better late than never, we'll hopefully get back on track soon. (And if you'd like to help contribute, progress report writers are always welcome!)
```

File: content/post/22-10-27-progress-report-37.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+{{< imgw "/images/post/pr_37/nukieplanet" "50%" >}}
+
+{{< newsection >}}
+### Stamina
+*Contributed by metalgearsloth*
+
+<!-- description doesnt make it very clear what was changed -->
+After a series of lawsuits, crewmembers are no longer perpetually amped up on epinephrine and amphetamines; instead, everyone is now susceptible to stamina damage!
+
+Work on your dodging skills or disabler beams will have you stunned in no time.
```
Comment (moonheart08) [inline content/post/22-10-27-progress-report-37.md:103]:
```suggestion
Work on your dodging skills or disabler beams will have you stunned in no time, but you're no longer prone to falling over from a single shot of a taser.
```

**Comment by moonheart08** [comment]
@ike709 no TODOs left, the date is correct. We should publish.

**Review by moonheart08 — APPROVED** [review]

**Pull request merged**